src/module.c

```diff
@@ -2205,6 +2205,18 @@ int RM_ReplyWithDouble(RedisModuleCtx *ctx, double d) {
     return REDISMODULE_OK;
 }
 
+/* Send a string as a BigNumber reply.  
+ * It does not appear that any verification is done on this string to ensure
+ * it is a valid BigNumber.  should it?
```

Comment:
@sjpotter This comment is used to auto-generate API documentation, so it can't be left in this state.